northstar/src/runtime/island/fs.rs

```diff
@@ -143,6 +146,7 @@ fn bind(root: &Path, target: &Path, host: &Path, options: &MountOptions) -> Vec<
             fstype: None,
             flags: MsFlags::MS_BIND | flags,
             data: None,
+            err_str: err_str.clone(),
```

Comment:
This cries for `Mount::new(target, source, type, flags, data)` that internally constructs `err_str`.